### ISSUE TITLE
Language preference 

### ISSUE THREAD

**Issue by abhi8686** [worknenjoy/truppie#105]
Description:
1: language preference should be session based, as every time we it changes to default portegues. 

Case : 
    steps to reproduce:: 
         1: root page, 
         2: change language 
         3: close the tab
         4: open root page 
         5: language changed to default. 

i know that website is based out of portugues, still 

Make it a enhancement

**Comment by alexanmtz** [comment]
Yeah @abhi8686, you're right. The whole website should have the english version reviewed and let's keep this issue to have this enhancement as part of multi language support

**Comment by wteophilo** [comment]
Hi!
I new here and solve this problem... can you send me how i send the pull request?

**Comment by alexanmtz** [comment]
I saw your pull request @wteophilo ! I will review it
